[2023-12-03T09:38:46.486+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:38:45.080919+00:00 [queued]>
[2023-12-03T09:38:46.523+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:38:45.080919+00:00 [queued]>
[2023-12-03T09:38:46.524+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2023-12-03T09:38:46.605+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): load_to_bq_task> on 2023-12-03 09:38:45.080919+00:00
[2023-12-03T09:38:46.616+0000] {standard_task_runner.py:57} INFO - Started process 592241 to run task
[2023-12-03T09:38:46.623+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'dag_SFCrimes_data_bqload', 'load_to_bq_task', 'manual__2023-12-03T09:38:45.080919+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/dag_sf_crimes_bqload.py', '--cfg-path', '/tmp/tmpjc4ygkk_']
[2023-12-03T09:38:46.627+0000] {standard_task_runner.py:85} INFO - Job 92: Subtask load_to_bq_task
[2023-12-03T09:38:46.793+0000] {task_command.py:410} INFO - Running <TaskInstance: dag_SFCrimes_data_bqload.load_to_bq_task manual__2023-12-03T09:38:45.080919+00:00 [running]> on host jarvis
[2023-12-03T09:38:47.307+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_SFCrimes_data_bqload' AIRFLOW_CTX_TASK_ID='load_to_bq_task' AIRFLOW_CTX_EXECUTION_DATE='2023-12-03T09:38:45.080919+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-03T09:38:45.080919+00:00'
[2023-12-03T09:41:08.361+0000] {logging_mixin.py:149} INFO - Loaded 130 rows into tbl_raw_crime
[2023-12-03T09:41:08.474+0000] {taskinstance.py:1847} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.10/dist-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/mani-dev/airflow/dags/dag_sf_crimes_bqload.py", line 42, in load_bq_raw
    delete_variable(key='SFcrimes_'+tail)
  File "/home/mani-dev/airflow/dags/utils/adminvars.py", line 17, in delete_variable
    deleted_variable = Variable.get(key=key)
  File "/usr/local/lib/python3.10/dist-packages/airflow/models/variable.py", line 142, in get
    raise KeyError(f"Variable {key} does not exist")
KeyError: 'Variable SFcrimes_20231203 does not exist'
[2023-12-03T09:41:08.517+0000] {taskinstance.py:1368} INFO - Marking task as UP_FOR_RETRY. dag_id=dag_SFCrimes_data_bqload, task_id=load_to_bq_task, execution_date=20231203T093845, start_date=20231203T093846, end_date=20231203T094108
[2023-12-03T09:41:08.602+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 92 for task load_to_bq_task ('Variable SFcrimes_20231203 does not exist'; 592241)
[2023-12-03T09:41:08.638+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 1
[2023-12-03T09:41:08.711+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
